examples/setup_recover.yml

```diff
@@ -0,0 +1,13 @@
+- name: Setup oVirt environment
+  hosts: localhost
+  connection: local
+  vars:
+     detach: "{{ clean | default(False) }}"
```

Comment:
Yes, good point, I will change it.
Actually I was thinking to change the value from boolean to the destination setup that the user will want to clean/recover.
Since there are 2 setups, primary oVirt setup and secondary oVirt setup, when the user will try fail over, in case of a disaster, from the primary to the secondary one will want to clean the primary setup, and when the user will want to fail back again to the primary setup, one will want to clean the secondary setup and recover back the primary setup.

so for example when we will want to call the playbook it thought to dosomething like this:
   ansible-playbook setup_recover.yml -i recovery.ini --extra-vars "recover=Primary clean=Secondary" --ask-vault-pass